doc/api/path.md

```diff
@@ -416,6 +416,16 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+### path.posix.normalize(path)
```

Comment:
Should I still give it a heading or would it be better as just a note above line 341 (about the TypeError)?